owncloudApp/src/main/res/values/strings.xml

```diff
@@ -597,5 +597,9 @@
     <string name="welcome_feature_7_text">Select all files at once</string>
     <string name="welcome_feature_8_title">Available offline</string>
     <string name="welcome_feature_8_text">Your files automatically synchronized</string>
+    <string name="welcome_feature_9_title">Document provider</string>
+    <string name="welcome_feature_9_text">Access and edit your ownCloud files from other apps</string>
```

Comment:
should be the `ownCloud` word brandable?

Reply:
Since the wizard is not brandable yet, I would not address this change here.